src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import duke.tasks.TaskList;
+
+/**
+ * CLI Task manager based on Duke.
+ *
+ * @author bhcs
+ */
+public class Duke {
+
+    private final Ui ui;
```

Comment:
I'm not too sure myself if **all** `final` variables have to be in caps (doesn't make sense to me either if it's not `static`) but for the record it is stated [here](https://se-education.org/guides/conventions/java/intermediate.html#naming) to be the case